another.yaml

```diff
@@ -1 +1,2 @@
 password=12345678
+password=9874352622
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:** password=**********  
[This comment was created by Aqua Pipeline]